<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1014</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionFit_To_Window"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionShow_Brief_Only"/>
    <addaction name="actionShow_Only_Missing"/>
    <addaction name="actionShow_Fake_Blocks_and_Arcs"/>
    <addaction name="actionShow_Demangle_Names"/>
    <addaction name="separator"/>
    <addaction name="actionProject_Directory"/>
    <addaction name="actionSource_Directories"/>
    <addaction name="actionBuild_Directories"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHow_To"/>
    <addaction name="actionAbout_Trucov"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionRun"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuOptions"/>
   <addaction name="menuRun"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionSave_Project_As">
   <property name="text">
    <string>Save Project As</string>
   </property>
  </action>
  <action name="actionSave_Report">
   <property name="text">
    <string>Save Report</string>
   </property>
  </action>
  <action name="actionSave_Report_As">
   <property name="text">
    <string>Save Report As</string>
   </property>
  </action>
  <action name="actionSave_All_Report">
   <property name="text">
    <string>Save All Reports</string>
   </property>
  </action>
  <action name="actionSave_Graph">
   <property name="text">
    <string>Save Graph</string>
   </property>
  </action>
  <action name="actionSave_Graph_As">
   <property name="text">
    <string>Save Graph As</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionFind_Again">
   <property name="text">
    <string>Find Again</string>
   </property>
  </action>
  <action name="actionControl_Flow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Control Flow</string>
   </property>
  </action>
  <action name="actionSource_Code">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Source Code</string>
   </property>
  </action>
  <action name="actionProject_Explorer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Project Explorer</string>
   </property>
  </action>
  <action name="actionHow_To">
   <property name="text">
    <string>How To</string>
   </property>
  </action>
  <action name="actionAbout_Trucov">
   <property name="text">
    <string>About Trucov</string>
   </property>
  </action>
  <action name="actionShow_Brief_Only">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Brief Only</string>
   </property>
  </action>
  <action name="actionShow_Only_Missing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Only Missing</string>
   </property>
  </action>
  <action name="actionShow_Fake_Blocks_and_Arcs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Fake Blocks and Arcs</string>
   </property>
  </action>
  <action name="actionShow_Demangle_Names">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Demangled Names</string>
   </property>
  </action>
  <action name="actionSource_Directories">
   <property name="text">
    <string>Source Directories</string>
   </property>
  </action>
  <action name="actionBuild_Directories">
   <property name="text">
    <string>Build Directories</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionFit_To_Window">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fit To Frame</string>
   </property>
  </action>
  <action name="actionProject_Directory">
   <property name="text">
    <string>Project Directory</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout_Trucov</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
